frontend/src/metabase/query_builder/components/NativeQueryEditor/CodeMirrorEditor/CodeMirrorEditor.module.css

```diff
@@ -0,0 +1,186 @@
+:global {
+  :local(.editor) {
+    width: 100%;
+    height: 100%;
```

Comment:
Editor is not vertically scrollable when content overflows, footer of SQL preview is not in viewport.

![image](https://github.com/user-attachments/assets/e4821028-ee9a-4dc1-8733-a89fbe8aed59)